.rubocop.yml

```diff
@@ -3,6 +3,19 @@ inherit_gem:
     - config/default.yml
     - config/rails.yml
 
+require:
+  - ./lib/rubocop/cop/govuk/govuk_button_to.rb
+  - ./lib/rubocop/cop/govuk/govuk_link_to.rb
+  - ./lib/rubocop/cop/govuk/govuk_submit.rb
+
+Govuk:
+  Include:
+    - "app/views/**/*"
+    - "app/components/**/*"
+  Exclude:
+    - "app/components/trainees/confirmation/degrees/view.html.erb"
```

Comment:
Please could these two exclusions also be carded up? To fix them, we need to tell the forms to use the GovUK form builder:

```
<%= form_with model: [trainee, degree], method: :delete, local: true, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
```

and

```
<%= form_with url: "/auth/developer/callback", builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
```

and then you should be able to use `f.govuk_submit` rather than `f.submit`.

Maybe there's a larger piece to see whether view component can be configured to use our own form builder by default? @gpeng

Reply:
https://trello.com/c/oLB4z56W/923-check-whether-view-component-can-be-configured-to-use-our-own-form-builder-by-default